MHIdentityKit/Infrastructure/IdentityManager/OAuth2IdentityManager.swift

```diff
@@ -203,6 +203,21 @@ open class OAuth2IdentityManager: IdentityManager {
         }
     }
     
+    @available(iOS 13, tvOS 13.0.0, macOS 10.15, *)
+    open func authorize(request: URLRequest, forceAuthenticate: Bool) async throws -> URLRequest {
```

Comment:
This means I have to delete the test function testOAuth2IdentityManager() in OAuth2IdentityManagerAsyncTests class since it used the async overload of authorize().

Reply:
No, because the same function exist as protocol extension on the `IdentityManager` - the tests should be using that.